Test/Directory.Build.props

```diff
@@ -27,8 +27,8 @@
     <!-- put obj files in a mirror tree off the root to keep the src folder tree cleaner -->
     <BaseIntermediateOutputPath Condition=" '$(BaseIntermediateOutputPath)' == '' ">$(EnlistmentRoot)\obj\$([MSBuild]::MakeRelative('$(EnlistmentRoot)\', $(MSBuildProjectDirectory)))</BaseIntermediateOutputPath>
 
-    <!-- Tests currently only work for netfx -->
-    <TargetFrameworks>$(NetfxVersion)</TargetFrameworks>
+    <!-- Test on netcore to cover all platforms -->
+    <TargetFrameworks>$(NetCoreVersion)</TargetFrameworks>
```

Comment:
Do we want to run tests for both target frameworks?

Reply:
yes, yes we do.  yikes - fixed